# HG changeset patch
# User Rik <rik@octave.org>
# Date 1412100030 25200
#      Tue Sep 30 11:00:30 2014 -0700
# Node ID d70b1cec774331bb69fb05ee6675a908211860ed
# Parent  c3611856cdd490967913e830bf3a550aa18a43ce
NEWS: Announce addition of XOR reduction to xor function.

* NEWS: Announce addition of XOR reduction to xor function.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -61,16 +61,19 @@ Summary of important user-visible change
  ** linkprop has been completely re-coded for performance and Matlab
     compatibility.  It now returns a linkprop object which must be stored
     in a variable for as long as the graphic objects should remain linked.
     To unlink properties use 'clear hlink' where hlink is the variable
     containing the linkprop object.
 
  ** isprime has been extended to operate on negative and complex inputs.
 
+ ** xor has been extended to accept more than two arguments in which case
+    it performs cumulative XOR reduction.
+
  ** The following functions now support N-dimensional arrays:
 
       fliplr   flipud   rot90
 
  ** The new warning ID "Octave:data-file-in-path" replaces the three
     previous separate warning IDs "Octave:fopen-file-in-path",
     "Octave:load-file-in-path", and "Octave:md5sum-file-in-path".
 
